scilpy/gradients/utils.py

```diff
@@ -3,10 +3,14 @@
 import numpy as np
 
 
-def random_uniform_on_sphere(nb_vectors):
+def random_uniform_on_half_sphere(nb_vectors):
```

Comment:
I don't think this produces points on the half sphere, but rather the whole sphere. I believe the previous name was right.

Reply:
Ok, I'll change back!

Here is the plot from running it a few times with 10 points.

![image](https://github.com/scilus/scilpy/assets/4967417/25baae40-0816-4926-96d6-5866968c5c6f)

Reply:
Not super well distributed, but not a half sphere